src/metabase/driver/crate.clj

```diff
@@ -4,7 +4,9 @@
             [metabase.driver :as driver]
             [metabase.driver.crate.util :as crate-util]
             [metabase.driver.generic-sql :as sql]
-            [metabase.util :as u]))
+            [metabase.util :as u]
+            [clojure.tools.logging :as log])
+  (:import (java.sql.DatabaseMetaData)))
```

Comment:
you can just do `(:import java.sql.DatabaseMetaData)` (no need to wrap in parens)